### PR TITLE
docs/p2p: updates from review

### PR DESCRIPTION
**Pull request by ebuchman**
Addresses feedback from https://github.com/tendermint/tendermint/pull/1057

### CODE REVIEW

File: docs/specification/new-spec/p2p/config.md [hunk 1]
```diff
@@ -6,14 +6,14 @@ Here we describe configuration options around the Peer Exchange.
 
 `--p2p.seed_mode`
 
-The node operates in seed mode. It will kick incoming peers after sharing some peers.
-It will continually crawl the network for peers.
+The node operates in seed mode. In seed mode, a node continuously crawls the network for peers,
+and upon incoming connection shares some peers and disconnect.
```
Comment (melekes) [inline docs/specification/new-spec/p2p/config.md]: `disconnects`

File: docs/specification/new-spec/p2p/peer.md [hunk 2]
```diff
@@ -86,14 +82,14 @@ The Tendermint Version Handshake allows the peers to exchange their NodeInfo:
...
-	Version    string        `json:"version"` // major.minor.revision
-    Channels   []int8        `json:"channels"` // active reactor channels
-	Other      []string      `json:"other"`   // other application specific data
+	PubKey     crypto.PubKey
+	Moniker    string
+	Network    string
+	RemoteAddr string
+	ListenAddr string
+	Version    string
+    Channels   []int8
```
Comment (melekes) [inline docs/specification/new-spec/p2p/peer.md]: something wrong with formatting

File: docs/specification/new-spec/p2p/connection.md [hunk 3]
```diff
@@ -1,25 +1,28 @@
...
 
 ### Ping and Pong
 
-The ping and pong messages consist of writing a single byte to the connection; 0x1 and 0x2, respectively
+The ping and pong messages consist of writing a single byte to the connection; 0x1 and 0x2, respectively.
 
 When we haven't received any messages on an `MConnection` in a time `pingTimeout`, we send a ping message.
-When a ping is received on the `MConnection`, a pong is sent in response.
+When a ping is received on the `MConnection`, a pong is sent in response only if there are no other messages
+to send and the peer has not sent us too many pings.
```
Comment (odeke-em) [inline docs/specification/new-spec/p2p/connection.md]: How many are `too many pings`?

Comment (melekes) [reply docs/specification/new-spec/p2p/connection.md]:
```
			// TODO: prevent abuse, as they cause flush()'s.
			c.Logger.Debug("Receive Ping")
			c.pong <- struct{}{}
```

it's a TODO I believe

**Pull request merged**